pyopenms_viz/_core.py

```
@@ -13,7 +13,8 @@
 from pandas.core.dtypes.common import is_integer
 from pandas.util._decorators import Appender
 import re
-
+from typing import Optional
```

Ruff (0.8.2)

Redefinition of unused `Optional` from line 4

Remove definition: `Optional`

(F811)